hazel.opam.locked

```diff
@@ -0,0 +1,198 @@
+opam-version: "2.0"
+name: "hazel"
+version: "~dev"
+license: "MIT"
+homepage: "https://github.com/hazelgrove/hazel"
+bug-reports: "https://github.com/hazelgrove/hazel/issues"
+depends: [
+  "abstract_algebra" {= "v0.16.0"}
+  "alcotest" {= "1.8.0" & with-test}
+  "angstrom" {= "0.16.0"}
+  "astring" {= "0.8.5"}
+  "async_js" {= "v0.16.0"}
+  "async_kernel" {= "v0.16.0"}
+  "async_rpc_kernel" {= "v0.16.0"}
+  "base" {= "v0.16.3"}
+  "base-bigarray" {= "base"}
+  "base-bytes" {= "base"}
+  "base-domains" {= "base"}
+  "base-nnp" {= "base"}
+  "base-threads" {= "base"}
+  "base-unix" {= "base"}
+  "base64" {= "3.5.1"}
+  "base_bigstring" {= "v0.16.0"}
+  "base_quickcheck" {= "v0.16.0"}
+  "bignum" {= "v0.16.0"}
+  "bigstringaf" {= "0.10.0"}
+  "bin_prot" {= "v0.16.0"}
+  "camlp-streams" {= "5.0.1"}
+  "chrome-trace" {= "3.16.0"}
+  "cmdliner" {= "1.3.0"}
+  "conf-bash" {= "1"}
+  "conf-gmp" {= "4"}
+  "conf-pkg-config" {= "3"}
+  "core" {= "v0.16.2"}
+  "core_kernel" {= "v0.16.0"}
+  "cppo" {= "1.6.9"}
+  "csexp" {= "1.5.2"}
+  "diffable" {= "v0.16.0"}
+  "dune" {= "3.16.0"}
+  "dune-build-info" {= "3.16.0"}
+  "dune-configurator" {= "3.16.0"}
+  "dune-rpc" {= "3.16.0"}
+  "dyn" {= "3.16.0"}
+  "either" {= "1.0.0"}
+  "ezjs_idb" {= "0.1.1"}
+  "ezjs_min" {= "0.3.0"}
+  "fiber" {= "3.7.0"}
+  "fieldslib" {= "v0.16.0"}
+  "fix" {= "20230505"}
+  "fmt" {= "0.9.0" & with-test}
+  "fpath" {= "0.7.3"}
+  "gen" {= "1.1"}
+  "gen_js_api" {= "1.1.2"}
+  "host-arch-x86_64" {= "1"}
```

Comment:
this is causing problems on my mac -- I think it works fine to remove this and update the `make change-deps` target with the sed command I had in there before to remove it

Reply:
I'll add the sed command and add another issue for tracking https://github.com/ocaml/opam/issues/6042

@cyrus- do you want me to see if we can have CI run for osxarm64 as well so I don't accidentally do this again?

Reply:
@7h3kk1d seems a bit wasteful to do that on every build since we don't actually have any native code being generated in Hazel, maybe just for pushes to `dev` so we can quickly correct if it happens again?

Reply:
https://github.com/hazelgrove/hazel/issues/1366